src/client/driver/index.js

```diff
@@ -173,6 +174,16 @@ export default class ClientDriver {
             });
     }
 
+    _onResizeWindowCommand (command) {
```

Comment:
we need to update document.title before resizing because `testcafe-browser-window` finds a browser window by it's title